packages/testing/src/fixture.ts

```diff
@@ -0,0 +1,188 @@
+import { existsSync } from 'fs'
+import { createRequire } from 'module'
+import { normalize } from 'path'
+import { fileURLToPath } from 'url'
+
+import test from 'ava'
+import cpy from 'cpy'
+import { execa, execaCommand } from 'execa'
+import stringify from 'fast-safe-stringify'
+import { getBinPathSync } from 'get-bin-path'
+import isPlainObj from 'is-plain-obj'
+
+import { createRepoDir, removeDir } from './dir.js'
+import { ServerHandler, startServer, Request } from './server.js'
+
+const require = createRequire(import.meta.url)
+
+export class Fixture {
+  flags: Record<string, unknown> = {
+    stable: true,
+    buffer: true,
+    branch: 'branch',
+    featureFlags: {},
+  }
+
+  env = {
+    // Ensure local environment variables aren't used during development
+    // TODO: check this one
```

Comment:
Do you plan to doing this before checking in?

Reply:
I'm needing this once I use the Fixture in the tests of `@netlify/build` but this will be done in a separate PR to keep them as small as possible (this one is already huge)